azure-mgmt-resources/src/main/java/com/microsoft/azure/management/resources/implementation/ResourceManager.java

```diff
@@ -181,6 +186,26 @@ public ResourceGroups resourceGroups() {
         return resourceGroups;
     }
 
+    /**
+     * @return the subscription management API entry point
+     */
+    public Subscriptions subscriptions() {
```

Comment:
Do we need to expose `subscriptions()` and `tenants()` here? The existing logic is like we make them available inside `Authenticated` as it above the level of resource management.

Reply:
Since it currently exposed in `Authenticated`, customer cannot use in the `Azure` object.

Reply:
We could expose it by save the `Authenticated` object, but I think use a new Client is more clear.